src/locales/en/product.json

```diff
@@ -4,9 +4,9 @@
     "pattern": "Pattern",
     "productDescription": {
         "description": "Product description",
-        "mainMaterial": "Main material",
-        "innerMaterial": "Inner material",
-        "bottomMaterial": "Bottom material",
+        "mainMaterial": "Main material - ",
```

Comment:
On my opinion - it's a bad idea to use separator in text-variables

Reply:
This text-variables that are used only once on the product page. Of course, this is not the best way, but I haven't found a way to put this separator in the description:
description={t('${bottomMaterial.material.translations_key}.name')}

when I write like this - ESLint blocks and reformats, and disabling and pushing it is also not the best idea, because when it wiil merge  and everyone will pull it ESLint immediately reformats
 description={ ' - ' + t(`${bottomMaterial.material.translations_key}.name`)}
 
 If you have any ideas or suggestions please write

Reply:
why u create var if u use it only once? maybe it require refactoring? don't scatter logic into small pieces!